src/macaron/slsa_analyzer/checks/check_result.py

```diff
@@ -29,6 +25,32 @@ class CheckResultType(str, Enum):
     UNKNOWN = "UNKNOWN"
 
 
+class Confidence(float, Enum):
+    """This class contains confidence score for a check result.
+
+    The scores must be in [0.0, 1.0].
+    """
+
+    #: A high confidence score.
+    HIGH = 1.0
```

Comment:
I know we have discussed that the values in this enum are not important. However, IMHO, if we include the value `1.0` i.e. 100%, then I would prefer the name `CERTAIN` over `HIGH` which better expresses that "this is 100% true". We can then have 4 values: `CERTAIN`, `HIGH`, `MEDIUM`, and `LOW`.